src/App.tsx

```diff
@@ -1,14 +1,62 @@
 /* eslint-disable max-len */
-import React from 'react';
+import {
+  useCallback,
+  useEffect,
+  useMemo,
+  useState,
+} from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
-import { TodoModal } from './components/TodoModal';
 import { Loader } from './components/Loader';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
+import { TodoModal } from './components/TodoModal';
+
+enum StatusTodos {
+  ACTIVE = 'active',
+  COMPLETED = 'completed',
+}
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [isTodosLoaded, setIsLoadTodos] = useState(false);
+  const [todoId, setTodoId] = useState(0);
+  const [status, setStatus] = useState('all');
+  const [query, setQuery] = useState('');
+
+  const loadTodos = useCallback(async () => {
+    const dataTodos = await getTodos();
+
+    setTodos(dataTodos);
+    setIsLoadTodos(true);
+  }, []);
+
+  useEffect(() => {
+    loadTodos();
+  }, []);
+
+  const getCheckQuery = useCallback((str: string) => {
+    return str.toUpperCase().includes(query.toUpperCase().trim());
```

Comment:
You can use implicit return if there is no additional lines in your arrow function